spinalcordtoolbox/deepseg/models.py

```diff
@@ -321,8 +395,23 @@
                              'a resolution of 0.05mm isotropic. Training data was provided by the Balgrist Center at'
                              'the University of Zurich.',
          'url': 'https://github.com/ivadomed/model_seg_mouse-sc_wm-gm_t1',
-         'models': ['model_seg_gm_wm_mouse_nnunet']},
-    'seg_sc_ms_lesion_stir_psir':
+         'models': ['model_seg_gm_wm_mouse_nnunet'],
+         'citation': textwrap.dedent("""
+             ```
+             @software{cohen_adad_2024_10819207,
+                       author={Cohen-Adad, Julien},
+                       title={{Segmentation model of ex vivo mouse spinal cord white and gray matter}},
```

Comment:
I'm surprised at the double-bracing here, which makes bibtex consider the entire title as escaped for upper-lower-casing.
```suggestion
                       title={Segmentation model of ex vivo mouse spinal cord white and gray matter},
```